packages/cardano-services/src/cli.ts

```diff
@@ -246,7 +252,18 @@ commonOptions(
       .default(DEFAULT_TOKEN_METADATA_CACHE_TTL)
       .argParser(cacheTtlValidator)
   )
-  .addOption(new Option('--use-queue', ProgramOptionDescriptions.UseQueue).env('USE_QUEUE').default(USE_QUEUE_DEFAULT))
+  .addOption(
+    new Option('--use-queue', ProgramOptionDescriptions.UseQueue)
+      .env('USE_QUEUE')
+      .default(USE_QUEUE_DEFAULT)
+      .argParser((useQueue) => {
+        stringToBoolean(
+          useQueue === undefined ? 'true' : useQueue,
+          Programs.HttpServer,
+          ProgramOptionDescriptions.UseQueue
+        );
+      })
+  )
```

Comment:
Are you sure this will work? I tried something similar but it requires a boolean

Reply:
This was quite a nasty bug, but I've resolved it in 8f92b7de7e37e651c2fb782c1d48f30bbc02c647